src/Http/Middleware/VerifyInternalApiToken.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace ThisIsDevelopment\LaravelApiTokenMiddleware\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+
+class VerifyInternalApiToken
+{
+    public function handle(Request $request, Closure $next): mixed
+    {
+        $headerToken = $request->header(config(
+            'laravel-api-token-middleware.token_header_key',
+            'X-API-Key'
```

Comment:
die default is niet nodig, met die config publish is het of default value daar of custom value in published app config